src/java.base/share/classes/java/util/jar/JarFile.java

```diff
@@ -152,6 +152,13 @@
     private static final boolean MULTI_RELEASE_ENABLED;
     private static final boolean MULTI_RELEASE_FORCED;
     private static final ThreadLocal<Boolean> isInitializing = new ThreadLocal<>();
+    /**
+     * The maximum size of array to allocate.
+     * Some VMs reserve some header words in an array.
+     * Attempts to allocate larger arrays may result in
+     * OutOfMemoryError: Required array size too large
+     */
+    private static final int MAX_BUFFER_SIZE = Integer.MAX_VALUE - 8;
```

Comment:
"/**" comments are generally only used for public documentation.  For use in this test case, probably a single line // comment would be sufficient to explain what this value is.

This constant is also named, "MAX_ARRAY_SIZE" in various places, which seems more applicable to this case.

Reply:
Hello Brent,

I've updated the PR with your suggested changes for this member variable name and the comment.